torchax/torchax/tensor.py

```diff
@@ -591,11 +574,20 @@ def to_tensor(x):
     return res
 
   def j2t_iso(self, jaxarray):
```

Comment:
Technically not this PR, but would you like to document `j2t_iso` and `j2t_copy`? It seems that `j2t_iso` gives you a view, while `j2t_copy` gives you a copy? But then all Jax arrays are immutable so I'm not sure what value does `j2t_copy` provide on top of `j2t_iso`. Would be good to clarify to the user.

Reply:
done.